.github/workflows/test.yml

```diff
@@ -81,6 +81,31 @@ jobs:
       - name: Integration Tests
         run: sbt integrationTests/test
 
+  std:
```

Comment:
Could you add the `std` job to the `needs` list of the `all_tests_passed` job below (~line 115).

Besides that I think this is a good idea.